### PR TITLE
chore(ci-cd): fix yarn lint

### PR DESCRIPTION
**Pull request by shunkakinoki**
# Checklist:

* [x] I have followed the guidelines in our [Contributing Guidelines](https://github.com/narative/gatsby-theme-novela/blob/master/CONTRIBUTING.md). _Especially our commitlint_
* [x] I have checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/fix/change.
* [x] I have checked for an open issue related to this. _There isn't one / There is one : #XXX_
* [x] I have performed a self-review of my own code.
* [x] I have performed the necessary tests locally.
* [x] I have linted my code locally prior to submission.
* [x] If applicable, I have commented my code, particularly in hard-to-understand areas

# Type of PR

* [x] Bug fix (_non-breaking change which fixes an issue_)
* [ ] New feature (_adding a feature following a feature-request issue_)
* [ ] Improvement (_improving an existing feature - includes `style:` and `perf:`commits_)
* [ ] Refactor (_rewriting existing code without any feature change_)
* [ ] (!) This change is or requires a documentation update

# Description

Fixes bug in `yarn lint` in 

"import/no-extraneous-dependencies" errors

Please include a summary of the change and what is changed by this PR. Please also include relevant motivation and context.

Seriously love your project. Never seen anything like it.
I'm just motivated to fix the CI/CD work pipeline so that the development to release process will become smooth (personally waiting for a new release)
Fixes coming to tackle the `yarn build` commands too.

## Additional information/context
_If relevant, add any information or context that would be useful to evaluate this PR_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,8 @@
             "kebabCase": true
           }
         }
-      ]
+      ],
```
Comment (aleccool213) [inline package.json]: Isn’t it better practice to install extraneous packages? Package managers like pnpm and yarn 2 enforce this.

Comment (shunkakinoki) [reply package.json]: @aleccool213 yes, you're completely right
Configured specific rules for an entire file through comments on the specific file on the new PR.

Comment (shunkakinoki) [reply package.json]: For now, although `yarn build` still fails,
`yarn lint` succeeds
hence the purpose of this PR

**Comment by shunkakinoki** [comment]
removed `yarn.lock` from the PR
`yarn lint` passes for now!
will work to fix the `yarn build` ci/cd

**Pull request closed without merging**